config.yml

```diff
@@ -1,3 +1,3 @@
 stats:
-  - report: true
-    url: https://metrics.devwithlando.io
+ - report: true
+   url: https://metrics.lando.dev
```

Comment:
Indent one more space for lines 2 and 3?

Reply:
👍